packages/ckeditor5-ckbox/tests/ckboxediting.js

```diff
@@ -78,13 +78,16 @@ describe( 'CKBoxEditing', () => {
 	it( 'should not register the "ckbox" command if CKBox lib is missing', async () => {
 		delete window.CKBox;
 
+		sinon.stub( console, 'warn' );
+
 		const editor = await createTestEditor( {
 			ckbox: {
 				tokenUrl: 'http://cs.example.com'
 			}
 		} );
 
 		expect( editor.commands.get( 'ckbox' ) ).to.be.undefined;
+		expect( console.warn.callCount ).to.equal( 1 );
```

Comment:
It would be nice, to check the warning text as well. I.e. if it contains `ckbox-library-not-loaded`.